learn/templates/base.html

```diff
@@ -7,7 +7,7 @@
 	{% endblock %}
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	
+	<link rel="shortcut icon" type="image/png" href="favicon.png">
```

Comment:
change href address to `"{% static 'images/favicon.png %}"`
ref - https://docs.djangoproject.com/en/2.1/howto/static-files/

Reply:
Ok I will edit it